src/twister2/yaml_test_specification.py

```diff
@@ -18,10 +17,11 @@ class YamlTestSpecification:
     """Test specification for yaml test."""
     name: str  #: test case name plus platform
     original_name: str  #: keeps test case name without platform
-    path: Path  #: path to a folder where C files are stored
+    source_dir: Path  #: path to a folder where C files are stored
     rel_to_base_path: Path  #: path relative to zephyr base
     platform: str  #: platform name used for this test
-    build_dir: Optional[Path] = None  #: path to dir with build results
+    build_name: str = ''  #: name of build configuration from yaml
+    output_dir: Path = Path('output_dir')  #: path to dir with build results
```

Comment:
I'm not sure if keeping here once again information about `output_dir` is a good idea. This information is kept in `TwisterConfig` (`src/twister2/twister_config.py`), and for me it is enough. Now for me it is confusing what `output_dir` really means in this context (and why default value is set as `'output_dir'`). If you really want to have this information here, then please change name for this variable as for example `general_output_dir` and change its description as `#: path to general Twister output directory`.

Reply:
I will change it to `Path('.')` . I need to have some default value and it must by `Path`.

Reply:
Can you change this default value to `Path('.')`?

Reply:
changed